libs/base/ui/form/password/src/password-input.component.ts

```diff
@@ -62,15 +64,15 @@ export class PasswordInputComponent
   @property({ type: Boolean }) visible?: boolean;
 
   @state()
-  minLengthError = false;
```

Comment:
I believe that this is not a component issue. Maybe better to hydrate UserRegistrationComponent on window:load and add a TODO comment to find better solution?

Reply:
Prehydration script is another method. Not sure though if we are happy with that solution over just hydrating on load.

Reply:
Should be addressed in a separate ticket. Hydrate on load for now.